drivers/platform/aducm3029/timer.c

```diff
@@ -110,16 +110,18 @@ static void tmr_callback(void *param, uint32_t tmr_event, void *arg)
  */
 int32_t timer_init(struct timer_desc **desc, struct timer_init_param *param)
 {
+	struct timer_desc *descriptor;
 	struct aducm_timer_desc *aducm_desc;
 	ADI_TMR_CONFIG		tmr_conf;
 
 	if (!desc || !param || param->freq_hz > FREQ_1MHZ)
 		return FAILURE;
 
-	if (!(*desc = calloc(1, sizeof(**desc))))
+	if (!(descriptor = calloc(1, sizeof(*descriptor)))) {
 		return FAILURE;
+	}
```

Comment:
{} are not needed.
Also, not related to your commit but if you can fix my nasty styling: 
```
ldesc =calloc(1, sizeof(*descriptor));
if (!ldesc)
      return FAILURE; 
```